src/libServer/EthRpcMethods.cpp

```diff
@@ -692,18 +694,20 @@ std::string EthRpcMethods::GetEthEstimateGas(const Json::Value& json) {
     std::swap(data, code);
   }
 
-  uint64_t gas = GasConv::GasUnitsFromCoreToEth(2 * DS_MICROBLOCK_GAS_LIMIT);
+  uint64_t gas = GasConv::GasUnitsFromCoreToEth(DS_MICROBLOCK_GAS_LIMIT / 2);
```

Comment:
some of the tests fail because they don't have enough funds to estimate this much gas - could/should this value be calculated as account max((account_balance - tx_value), DS_MICROBLOCK_GAS_LIMIT)?